src/workerd/io/compatibility-date.capnp

```diff
@@ -504,4 +504,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("legacy_module_registry")
       $experimental;
   # Enables of the new module registry implementation.
+
+  kvDirectBinding @53 :Bool
+      $compatEnableFlag("kvDirectBinding")
```

Comment:
Please try to match the naming convention used by all other compat flags in this file.

Reply:
Done.